README.md

```diff
@@ -28,9 +28,15 @@ Among the features that `Richie` offers out of the box:
 
 ## Quick preview
 
-If you're just looking for a quick preview of `Richie`, you can take look and have a tour of `Richie` on our dedicated [demo site](https://demo.richie.education).
+If you're just looking for a quick preview of `Richie`, you can take look and have a tour of
+`Richie` on our dedicated [demo site](https://demo.richie.education).
 
-Login/password are `admin`/`admin`. The database is regularly flushed.
+It is connected back-to-back with a demo of OpenEdX running on
+[OpenEdX Docker](https://github.com/openfun/openedx-docker).
+
+Login/password are `admin@example.com`/`admin`.
```

Comment:
You can also add the regular user `edx@example.com` / `edx`

Reply:
<img src="https://media.giphy.com/media/iurIHLBxms7UQ/giphy-downsized.gif" />

Reply:
and now I'm wondering why we called it edx and not student :man_shrugging: